openaerostruct/docs/advanced_features/multi_section_surfaces.rst

```diff
@@ -125,4 +123,50 @@ The steps required to setup the multi-section geometry group by constraint featu
    :end-before: checkpoint 4
 
 
+Multi-section surfaces for aerostructural problems
```

Comment:
Since now we have aerostructural, should we change line 3 to "Multi-section Surfaces for aerodynamic problems" or "VLM problems" or something like that?